client/InvoiceTickets.js

```diff
@@ -10,10 +22,38 @@ Template.InvoiceTickets.helpers({
     return InvoiceTickets.byTimeRange(
       FlowRouter.getParam("filter"),
       FlowRouter.getQueryParam("sortBy"),
-      FlowRouter.getQueryParam("sortOrder")
+      FlowRouter.getQueryParam("sortOrder"),
+      Session.get("itemsLimit")
     );
   },
   formatDate: function(date){
     return moment(date).format("MM-DD-YYYY");
   },
 });
+Template.InvoiceTickets.moreResults = function() {
+    // If, once the subscription is ready, we have less rows than we
+    // asked for, we've got all the rows in the collection.
+    return !(InvoiceTickets.find().count() < Session.get("itemsLimit"));
```

Comment:
This won't work :wink: that's because on the client you will not have all invoice tickets loaded (that's the reason we are doing the incremental loading in the first place!). To get the total count of invoice tickets, there are no easy / good ways – calling a method and returning the count from the server-side is still the easiest (although not fully reactive).